docker/README

```diff
@@ -0,0 +1,20 @@
+These Dockerfiles will create a DronecodeSDK development container image based on whichever flavour of Ubuntu or Fedora you prefer.
+
+To create an image :-
+
+cd /wherever/DronecodeSDK/docker
+docker build -t dronecode -f Dockerfile-Ubuntu-18.04 .
+
+To use the image :-
+
+cd /wherever/DronecodeSDK
+./run-docker
+
+This will create and run a container based on your image.
+/path/DronecodeSDK will be mounted inside the container at /home/dronecode/DronecodeSDK
+
+dronecode@dronecode-docker:~/DronecodeSDK$ ls
```

Comment:
This `ls` output will be out of sync quickly, I'd rather not commit it here.

Reply:
ok